src/careamics/config/callback_model.py

```diff
@@ -36,7 +36,7 @@ class CheckpointModel(BaseModel):
     save_last: Optional[Literal[True, False, "link"]] = Field(default=True)
     """When `True`, saves a last.ckpt copy whenever a checkpoint file gets saved."""
 
-    save_top_k: int = Field(default=3, ge=-1, le=100)
+    save_top_k: int = Field(default=1, ge=-1, le=100)
```

Comment:
I actually set to top 3 on purpose, since that is something people enjoy. I'd keep it and write a note stating that it is different from the defaults.

Reply:
Do you mean that it conflicts with `max_steps` or `limit_train_batches`? Because in truth, we don't care about `max_steps` and we care about saving multiple "best" checkpoints.